internal/pipeline/steps.go

```diff
@@ -142,11 +142,6 @@ func (s Steps) sign(env map[string]string, key jwk.Key) error {
 	for _, step := range s {
 		switch step := step.(type) {
 		case *CommandStep:
-			if step.Matrix != nil {
-				// Don't sign matrix steps... yet
-				continue
-			}
-
```

Comment:
Maybe do this in another PR?

Reply:
oh yes, definitely, that slipped in. will remove.